src/controllers/comment.controller.js

```diff
@@ -0,0 +1,56 @@
+import commentService from '../services/comment.service';
+import response from '../helpers/response.helper';
+import Paginate from '../helpers/paginate.helper';
+
+
+/**
+ * This class will controll all action related to the
+ * comment
+ */
+class comment {
+  /**
+     * This method will help
+     * to comment to the trip request
+     * @param {Object} req request data
+     * @param {Object} res response data
+     * @returns { Object} return a user message
+     */
+  static async createTripRequestComment(req, res) {
+    const Id = req.params.tripRequestID;
+    const result = await commentService.getComments(req, Id);
+    if (result) return response.successMessage(res, 'comment created successfuly', 201);
```

Comment:
@shemaeric1234 Always, it is considered to be best practice to return the entire object that got created.